src/integrations/Twilio.ts

```diff
@@ -0,0 +1,66 @@
+import {IntegrationName, IntegrationPurpose, IQualifierIntegration, IVerificationContext} from '../types';
+
+import { IObserverIntegration } from './../types';
+import Integration from "./Integration";
+
+export interface ITwilioConfig {
+  serviceSid: string;
+  authToken: string;
+}
+
+export interface ITwilioProps {
+  url: string;
+  config: ITwilioConfig;
+  requestId: string;
+}
+
+export default class TwilioIntegration extends Integration implements IQualifierIntegration{
+  twilioClient: any = null;
+  
+  constructor({ url, config, requestId }: ITwilioProps) {
+    super({
+      url,
+      config,
+      name: IntegrationName.MFA_TWILIO,
+      purposes: [IntegrationPurpose.QUALIFY],
+      requestId
+    });
+    console.log('twilio constructor', url, config, requestId);
+  }
+
+  public async configure() {
+    console.log('twilio config');
+    const _user_id = ''; // Set to the user's ID, username, or email address, or '' if not yet known.
+    const _session_id = this.dodgeballRequestId; // Set to a unique session ID for the visitor's current browsing session.
+
+    this.twilioClient = (window as any)._twilio = (window as any)._twilio || [];
+    this.twilioClient.push(["_setAccount", this.config.serviceSid]);
+    this.twilioClient.push(["_setUserId", _user_id]);
+    this.twilioClient.push(["_setSessionId", _session_id]);
+    this.twilioClient.push(["_trackPageview"]);
+  }
```

Comment:
This code looks like it was taken from the Sift integration, which is an observer, not a qualifier. I don't think this code is correct / configures the twilio client library correctly.